src/analysis/special_functions/pow.lean

```diff
@@ -1442,6 +1442,16 @@ begin
   rw [←nnreal.coe_rpow, real.to_nnreal_coe],
 end
 
+lemma eventually_pow_one_div_le (x : ℝ≥0) {ε : ℝ≥0} (hε : 1 < ε) :
```

Comment:
Here and in the next lemma, it sounds really weird to call `ε` a number that needs to be large enough. Could you change the notation so that the statement becomes easier to parse?

Reply:
sure, at least the hypothesis wasn't `ε < 0`! :stuck_out_tongue_winking_eye: